sources/packages/web/src/types/contracts/student/ApplicationContract.ts

```diff
@@ -136,7 +136,7 @@ export interface ApplicationSummaryDTO {
   studyEndPeriod: string;
   id: number;
   applicationName: string;
-  award: string;
+  submitted: string;
```

Comment:
is this renamed, due to the sort column name in the vue page?

Reply:
no, `award` is no longer in the application data table, instead, a new field called `submitted` is introduced